compiler/rustc_infer/src/infer/error_reporting/mod.rs

```diff
@@ -139,7 +139,7 @@ pub struct TypeErrCtxt<'a, 'tcx> {
 }
 
 impl<'a, 'tcx> TypeErrCtxt<'a, 'tcx> {
```

Comment:
Hmm, hard to improve on `'a` for `TypeErrCtxt` due to it being used in multiple fields.